docker-compose-production.yml

```diff
@@ -0,0 +1,58 @@
+
+version: '3'
+services:
+  postgres:
+    image: postgres:11
+    ports:
+      - "5432:${DB_PORT}"
+    environment:
+      POSTGRES_USER: ${DB_USERNAME}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: ${DB_DATABASE}
+    networks:
```

Comment:
would drop the separate networks, all containers run on a single sandboxed node during the build so there's no particular advantage to isolating them further